### PR TITLE
Import 1

### PR DESCRIPTION
**Pull request by hardik-surana**

### CODE REVIEW

File: public/ide/index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <title>GenXapp</title>
 </head>
 <body>
-    <div ng-controller="HeaderController" ng-include src="'views/header/header.view.html'">
+    <div ng-controller="HeaderController" ng-include src="'views/header/website.view.html'">
```
Comment (nischaym) [inline public/ide/index.html]: this should be header view... not the website i believe

File: public/ide/views/page/page.view.html [hunk 2]
```diff
@@ -1 +1,127 @@
-<h1>page</h1>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <script
```
Comment (nischaym) [inline public/ide/views/page/page.view.html]: you need to load these in the index.html

File: public/ide/views/page/page.view.html [hunk 3]
```diff
@@ -1 +1,127 @@
...
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" rel="stylesheet"></script>
+    <link href="../../css/bootstrap.min.css" rel="stylesheet"/>
+    <link href="../../css/styles.css" rel="stylesheet"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="UTF-8">
+    <title>Page List</title>
+</head>
+<body>
+
+<nav class="navbar navbar-default navbar-fixed-top">
```
Comment (nischaym) [inline public/ide/views/page/page.view.html]: only from here it should be present in the views page

File: public/ide/views/website/website.view.html [hunk 4]
```diff
@@ -1 +1,125 @@
-<h1>website</h1>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet"/>
+    <link href="../css/styles.css" rel="stylesheet"/>
```
Comment (nischaym) [inline public/ide/views/website/website.view.html]: These should go to index.html

Comment (anupmak) [reply public/ide/views/website/website.view.html]: its a common code and should sit in index,html.

File: public/ide/views/page/page.view.html (hunk 2 above)
Comment (anupmak) [reply public/ide/views/page/page.view.html]: common code should sit in index,html. So i guess it should be moved to index.html.

**Pull request merged**